go.mod

```diff
@@ -127,6 +127,7 @@ require (
 	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gvisor.dev/gvisor v0.0.0-20231023213702-2691a8f9b1cf // indirect
+	inet.af/tcpproxy v0.0.0-20221017015627-91f861402626 // indirect
```

Comment:
- https://github.com/containers/gvisor-tap-vsock/pull/399

Reply:
We could redirect the module in our `go.mod` file until `gvisor-tap-vsock` has a new release.

Reply:
`replace inet.af/tcpproxy => github.com/inetaf/tcpproxy v0.0.0-20240214030015-3ce
58045626c` doesn't seem to work?

```
go: github.com/inetaf/tcpproxy@v0.0.0-20240214030015-3ce58045626c used for two different module paths (github.com/inetaf/tcpproxy and inet.af/tcpproxy)
```